sycl/test-e2e/ESIMD/vadd_raw_send.cpp

```diff
@@ -8,7 +7,7 @@
 //===----------------------------------------------------------------------===//
 // REQUIRES: gpu-intel-gen9
 // UNSUPPORTED: gpu-intel-gen9 && windows
-// UNSUPPORTED: gpu-intel-dg1,gpu-intel-dg2,cuda,hip
+// UNSUPPORTED: gpu-intel-dg1,gpu-intel-dg2,cuda,hip,gpu-intel-pvc
```

Comment:
Sorry, but I don't fully understand the idea of this patch and the description does not help - it is empty.
Why did the test fail? Is it because 'REQUIRES: gpu-intel-gen9' is ignored on PVC?

If raw_send() API doesn't work on PVC, but tfloat32 doesn't work on GEN9, then we could just remove tfloat32 case - it doesn't give much additional coverage after we have got a test for sycl::half (that requires special handling of raw_t, etc).

If guesses above are correct, then the test can simply remove tfloat32 type/case, but keep sycl::half case. The new compile-only test is not so needed in this case.

Reply:
If you think it is a GPU RT issue, then please comment out sycl::half case and add a TODO comment asking to enable it when GPU RT is fixed. Please also create a tracker for GPU. 
If ESIMD FE compiles the code successfully, but it is wrong, then please create a tracker on ESIMD.

We announced that we implemented sycl::half in raw_send(), thus e2e test should be passing.

Alternatively, this test could be changed like this:
// RUN: %clangxx -fsycl %s -o %t1.out
// RUN: %GPU_RUN_PLACEHOLDER %t1.out
// RUN: %clangxx -fsycl -DNEW_API %s -o %t2.out
// RUN: %GPU_RUN_PLACEHOLDER %t2.out

// RUN: %clangxx -fsycl -DNEW_API -DTEST_SYCL_HALF_CASE %s -o %t3.out
// TODO: Enable this test when GPU RT is fixed.
// RUNx: %GPU_RUN_PLACEHOLDER %t3.out

Reply:
i think if you dump the gpu assembly, the .asm file (and not the .visaasm) has the actual instruction instread of the raw send